### PR TITLE
terminal: improvements and cleanup

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: src/lxc/terminal.c [hunk 1]
```diff
@@ -206,7 +206,10 @@ void lxc_terminal_signal_fini(struct lxc_terminal_state *ts)
 	if (ts->sigfd >= 0) {
 		close(ts->sigfd);
 
-		if (sigprocmask(SIG_SETMASK, &ts->oldmask, NULL) < 0)
+		int ret;
```
Comment (hallyn) [inline src/lxc/terminal.c]: So...  I'm pretty sure this was in our codingstyle, but do we really want to do this?  It's longer and arguably not clearer.

I'll merge it if you really want, but I prefer the old style here.

Comment (brauner) [reply src/lxc/terminal.c]: No, I can just remove it.

Comment (brauner) [reply src/lxc/terminal.c]: removed :)

**Comment by hallyn** [comment]
Thanks - I didn't look at every commit, but looks very nice.  Will merge when tests are done.

**Pull request merged**